setup.cfg

```diff
@@ -5,10 +5,14 @@ ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, E203
 exclude =
   bin,
   doc/conf.py,
-  **/*/__init__.py,
-  **/*/version.py,
+  **/__init__.py,
+  **/version.py,
   tests/.tests
+  tests/test_docs.py
 
 [tool:pytest]
 addopts = --flake8
 flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503 E203
```

Comment:
If you are using black you should only need this list to have `W503 E203` (not including the N* ones). Same for the line above that I can't comment on. This might resolve your flake8 issues.

Reply:
Thanks, but the flake8/black conflict arose because I had an array sliced as `data[indices,]`, where indices is a `tuple`. This used it to index the first dimension instead of trying to use an index for each dimension, but black wanted no space after the comma and flake8 wanted a space. My solution was to just reformat as `data[indices, :]`, which is probably more clear anyway.

Reply:
I still think you need to trim the flake8 ignore line to remove the irrelevant entries.

Reply:
Sorry, I didn't totally understand the sign of the N* entries in your comment above. So you're saying that I only need `flake8-ignore = W503 E203` since black is being run?

Reply:
I'm saying you don't need those first three E ignores. The N is entirely up to you since you are allowed to follow PEP variable naming in your package.